patroni/postgresql/bootstrap.py

```diff
@@ -280,22 +288,21 @@ def clone(self, clone_member):
         ret = self.create_replica(clone_member) == 0
         if ret:
             self._post_restore()
-            self._postgresql.configure_server_parameters()
         return ret
 
     def bootstrap(self, config):
         """ Initialize a new node from scratch and start it. """
         pg_hba = config.get('pg_hba', [])
         method = config.get('method') or 'initdb'
-        self._running_custom_bootstrap = method != 'initdb' and method in config and 'command' in config[method]
-        if self._running_custom_bootstrap:
+        if method != 'initdb' and method in config and 'command' in config[method]:
```

Comment:
so if one of the required params for a custom bootstrap method is not specified we fall back to `initdb` ? [docs](https://github.com/zalando/patroni/blob/master/docs/replica_bootstrap.rst#bootstrap) are unclear on that part 
> Each bootstrap method must define at least a name and a command. A special initdb method is available to trigger the default behavior, in which case method parameter can be omitted altogether.

Reply:
If there is no method defined we default to initdb.
If the method doesn't have a configuration section - default to initdb
If the configuration section doesn't have a command defined - default to initdb